### PR TITLE
docs: update installation, examples and production disclaimer

### PR DESCRIPTION
**Pull request by virkt25**
connected to #982 

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] Related API Documentation was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `packages/example-*` were updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ LoopBack 4 is a work in progress, the public API is frequently changed in
 backwards-incompatible ways. See [Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
 for more details.
 
+_LB4 is recommended for use to learn the latest and greatest [features and technologies](https://loopback.io/doc/en/lb4/Crafting-LoopBack-4.html).  However, it is not recommended for production use yet. For production use, we recommend using [LoopBack 3](https://loopback.io/doc/en/lb3/index.html)._
```
Comment (dhmlau) [inline README.md]: I wonder if it is better for rewording the first sentence as something like:
To learn about the latest and greatest LoopBack features and technologies for your next project, LoopBack 4 is recommended.

Comment (raymondfeng) [reply README.md]: We don't use terms such as `LB4` externally.

File: README.md [hunk 2]
```diff
@@ -15,43 +15,23 @@ LoopBack 4 is a work in progress, the public API is frequently changed in
 backwards-incompatible ways. See [Upcoming-Releases on wiki](https://github.com/strongloop/loopback-next/wiki/Upcoming-Releases)
```
Comment (raymondfeng) [inline README.md]: Should it be `backward incompatible`?

**Comment by virkt25** [comment]
@slnode test please

**Comment by virkt25** [comment]
@slnode test please

**Comment by virkt25** [comment]
@slnode test please

**Pull request merged**